### DISCUSSION TITLE
Bottom Toolbar and ...

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#915] [Ideas]
I don't know if you can implement changes on interface, but an option to move the toolbar bottom just like in kiwi browser would be great. Also, changing the location of the new tab button (+) to make the browser more convenient to use one handed.
Thanks for the browser 🖤

![mc-Yeni Bit Eşlem Resmi](https://user-images.githubusercontent.com/76991571/107204002-39b1ba00-6a0d-11eb-862a-95cbe539ddaf.jpg)

**Comment by ghost** [comment]
There was a feature on chrome://flags that could do that, but unfortunately was removed in some version (I don't remember what it was).

Yes, Kiwi can make the bar at bottom because it use a older version of Chromium.

I also want this on modern Bromite (Currently, Firefox it's the only modern browser that I know that support bottom bar).

**Reply by ghost** [reply]
Thanks, didn't know it was due to chromium.

**Reply by ghost** [reply]
Btw, I have just found out Vivaldi also supports it.

**Reply by sagarkpanda** [reply]
> There was a feature on chrome://flags that could do that, but unfortunately was removed in some version (I don't remember what it was).
> 
> Yes, Kiwi can make the bar at bottom because it use a older version of Chromium.
> 
> I also want this on modern Bromite (Currently, Firefox it's the only modern browser that I know that support bottom bar).

Now Kiwi is on v96 and still has the bottom toolabar. I also came here for the same but I see the answer now :)

**Comment by csagan5** [comment]
No, we do not develop or maintain UI changes for Bromite, it is out of scope for the project.

**Reply by ghost** [reply]
Thanks for informing.

**Comment by raQai** [comment]
It seems to be back 🤷 
I did not see the PR for this on chromium and only discovered this randomly by setting up a new device but:

```
Bromite 101.0.4951.53
Settings > Accessibility > Move toolbar to bottom
```

If anyone could link the PR/commit for this, I'd appreciate it 👍 I did not find it going through the chromium commits...

Looks like this one enabled it in bromium: https://github.com/bromite/bromite/pull/1975

**Reply by sagarkpanda** [reply]
https://github.com/bromite/bromite/pull/1975
